### PR TITLE
Jha 530

### PR DESCRIPTION
**Pull request by jlangy**
### Please check if the PR fulfills these requirements:

- [x] After these changes, the app was run and still works as expected
- [x] Unit tests for these changes were added (if applicable)
- [x] All existing unit tests were run and still pass

### Please specify the type of change this PR introduces (Bug fix, feature addition, content update, chore, etc.):
feature addition
### Additional Notes:

The general requirements for this ticket were:
- Allow power of attorney doc uploads in one file uploader on consent page
- Make a copy of uploaded PoA documents for each application (MSP, SB, or SuppBen) with unique uuids
- Send each copy to maxhub
- Include attachments in final application payload, in relevant subsection

### CODE REVIEW

**Review by bryce-mcmath — APPROVED** [review]
This looks great, love the tests. Assuming you got that new docType from Gerry

File: jha/tests/unit/services/apiService.spec.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+apiService._sendPostRequest = jest.fn(() => Promise.resolve(true));
+
+describe("Power Of Attorney", () => {
+  dummyData.hasPowerOfAttorney = true;
+  dummyData.fpcPowerOfAttorneyDocuments = [mockFile];
+
+  const getApplicationAttachments = async (data) => {
+    await apiService.sendApplication(dummyData);
+    let [, , postedFormData] = apiService._sendPostRequest.mock.calls[0];
```
Comment (bryce-mcmath) [inline jha/tests/unit/services/apiService.spec.js:13]: I've never seen this before, is this array destructuring and ignoring the first two values?

Comment (jlangy) [reply jha/tests/unit/services/apiService.spec.js:13]: yes, you can just put commas if you only need the nth value

Comment (jlangy) [reply jha/tests/unit/services/apiService.spec.js:13]: And yes that's right The PowerOfAttorney doctype was a request from kevin for maxhub

**Pull request merged**